run_robot_test.sh

```diff
@@ -1,14 +1,18 @@
-#/bin/bash
+#!/usr/bin/env bash
 
-TEST_CASE_FILE=tests/Tests
+# -------- Global Variables --------
+TEST_CASE_LOCATION=Tests
 TEST_VARIABLES_FILE=test-variables.yml
 TEST_VARIABLES=""
 TEST_ARTIFACT_DIR="test-output"
 EXTRA_ROBOT_ARGS=""
-SKIP_PIP_INSTALL=0
 TEST_INCLUDE_TAG=""
 TEST_EXCLUDE_TAG=""
+SKIP_PIP_INSTALL=0
 
+<<comment
+:'
+Command Line Parsing as implemented by ODS-ci
```

Comment:
Shouldn't it be `RHODA-CI`?

Reply:
The command line parsing code starting line 16 was implemented by ODS-ci, and as it was commented and preserved, I had added a this indicating that this is the original code of ODS-ci

well, anyways as we decided to remove all unused comments, this will get removed automatically